### PR TITLE
Windows build + maintainers

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by npavlovikj** [comment]
@SylvainCorlay , we couldn't make the Windows build work earlier. For some reason, the "library" directory is not created. Here is the thread for the issue you may find useful, https://github.com/openexr/openexr/issues/347#issuecomment-439736120. I tried the accepted suggestion, but that didn't work for me, and I haven't had much time to spend on it since.

**Comment by SylvainCorlay** [comment]
> @SylvainCorlay , we couldn't make the Windows build work earlier. For some reason, the "library" directory is not created. Here is the thread for the issue you may find useful, openexr/openexr#347 (comment). I tried the accepted suggestion, but that didn't work for me, and I haven't had much time to spend on it since.

Thanks for the pointer. The build on the QuantStack channel which was from the tag archive works, so I might fallback to it should this fail on windows.

**Comment by npavlovikj** [comment]
Also, I am not sure if this helps, but this is the thread from the initial "ilmbase" PR, and you can see all the Windows combinations Marcelo has tried, https://github.com/conda-forge/staged-recipes/pull/6785#issuecomment-428763887, including `skip: true  # [win and vc<14]`.

**Comment by SylvainCorlay** [comment]
@npavlovikj this appears to be OK now.

ilmbase's cmake is really not great...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,48 +12,47 @@ source:
...
     - {{ compiler('cxx') }}
     - cmake  # [win]
-    - ninja  # [win]
 
   host:
     - zlib
 
-  run:
-    - zlib
-
```
Comment (SylvainCorlay) [inline recipe/meta.yaml]: zlib has a run_exports section so it is not necessary to repeat it from the host section.

**Comment by SylvainCorlay** [comment]
@npavlovikj this is ready to go!

**Comment by SylvainCorlay** [comment]
Up! I am trying to get ilmbase <- openexr <- openimageio <- xtensor-io in, and I depend on this getting in!

**Comment by npavlovikj** [comment]
@SylvainCorlay , thanks for your work on this! Should I go ahead and merge the PR? You don't have permissions to do that?

**Comment by SylvainCorlay** [comment]
If it looks ok to you!

**Comment by npavlovikj** [comment]
Done :)

**Pull request merged**